src/Components/StockTrading.jsx

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable react/prop-types */
+import { Flex, Text } from "@sparrowengg/twigs-react";
+
+const StockTrading = ({ icon, title, borderBottom, bgColor }) => {
+  return (
+    <Flex
+      css={{
+        width: "250px",
+        paddingLeft: "10px",
```

Comment:
@9xVibee this can be $5, can you check the entire pr once there are many places which you can avoid using pixels